src/data/queries/tests/__snapshots__/newsStory.test.tsx.snap

```diff
@@ -121,18 +121,18 @@ Object {
   ",
   "date": "2019-05-14T15:35:12+00:00",
   "id": "bb4dcfc1-736b-4a9e-aa48-23c2dd4a4980",
-  "image": Object {
-    "alt": undefined,
-    "height": undefined,
-    "id": "c85b7882-f035-4b55-8b71-fcf81fade350",
-    "imageStyle": "full_width_content",
-    "styles": Array [
-      null,
-    ],
-    "title": undefined,
-    "url": "undefined",
-    "width": undefined,
-  },
+  "image": Array [
+    Object {
+      "alt": undefined,
+      "height": undefined,
+      "id": "5aa01b90-7156-45b1-85fb-95f82465409c",
+      "link": null,
+      "title": undefined,
+      "type": "media--image",
+      "url": undefined,
+      "width": undefined,
+    },
+  ],
```

Comment:
These null values are a result of the news story mock data not having the expanded field_media relationship